plugins/modules/cmci_create.py

```diff
@@ -26,9 +27,11 @@
 options:
   create_parameters:
     description: >
-      A list of one or more parameters that control the I(create) operation. Eligible parameters for the CREATE operation
-      can be found in the resource table reference for the target resource type, as listed in the CREATE operation
-      section of the "Valid CPSM operations" table. For example, the valid parameters for a PROGDEF CREATE operation
+      A list of one or more parameters that control the I(create) operation.
+      Eligible parameters for the CREATE operation can be found in the
+      resource table reference for the target resource type, as listed in the 
+      CREATE operation section of the "Valid CPSM operations" table. 
+      For example, the valid parameters for a PROGDEF CREATE operation
       are CSD and RESGROUP, as found in the L(PROGDEF resource table reference,
```

Comment:
Should probably put this link on its own line

Reply:
As discussed in Slack, breaking the linktext won't affect the link being rendered correctly.